doc/stdlib/math.rst

```diff
@@ -1348,7 +1446,18 @@ Mathematical Functions
 
    Take the inverse of ``x`` modulo ``m``\ : ``y`` such that :math:`x y = 1 \pmod m`\ , with :math:`div(x,y) = 0`\ . This is undefined for :math:`m = 0`\ , or if :math:`gcd(x,m) \neq 1`\ .
 
-.. function:: powermod(x, p, m)
+   .. doctest::
+
+       julia> invmod(2,5)
+       3
+
+       julia> invmod(2,3)
+       2
+
+       julia> invmod(5,6)
+       5
+
+.. function:: powermod(x::Integer, p::Integer, m)
 
    .. Docstring generated from Julia source
 
```

Comment:
Doesn't render in my OSX terminal. But I think the `modulo` earlier in the docstring is enough of a hint.